apps/src/StudioApp.js

```diff
@@ -1115,6 +1116,7 @@ StudioApp.prototype.showInstructionsDialog_ = function (level, autoClose) {
         <DialogInstructions />
       </Provider>,
       instructionsReactContainer);
+    resetAniGif(this.instructionsDialog.div.find('img.aniGif').get(0));
```

Comment:
`instructionsDialog.div` is a jquery object presumably?

Reply:
Yep: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/LegacyDialog.js#L61